docs/source/markdown/podman-run.1.md

```diff
@@ -893,6 +893,7 @@ Security Options
 - **no-new-privileges**: Disable container processes from gaining additional privileges
 - **seccomp=unconfined**: Turn off seccomp confinement for the container
 - **seccomp**=_profile.json_: Allowed syscall list seccomp JSON file to be used as a seccomp filter
+- **systempaths=unconfined**: Unmask all the paths that as masked by defaults. This is the same as **unmask=ALL**.
```

Comment:
No need to document it, I would just leave it undocumented,  Since we have the greater feature.  This is only there for docker compatibility.

BTW Any time you update the -run man page, you probably need to update the -create man page.

Reply:
Removed from the doc and update the podman create doc as well.